src/components/global/Navbar.jsx

```diff
@@ -1,8 +1,147 @@
+  right: 0,
+  top: -50,
+  height: '250px',
+  backgroundImage: { md: 'url(/assets/logoBlobNav.svg)', sm: 'none' },
+  backgroundRepeat: 'no-repeat',
+  display: {
+    lg: 'flex',
+    xs: 'none'
+  },
+  maxWidth: '200px'
+};
+
+const textStyle = {
+  textDecoration: 'none',
+  color: 'inherit'
 };
 
-export default Navbar;
+function NavBar() {
+  const [darkMode, setDarkMode] = useState(false);
+  const [menuOpen, setMenuOpen] = useState(false);
+
+  // anchor for menu to pop from its icon
+  const anchorRef = useRef(null);
+
+  const handleThemeChange = () => {
+    setDarkMode(!darkMode);
+  };
+
+  const handleToggle = () => {
+    setMenuOpen((prevOpen) => !prevOpen);
+  };
+
+  const handleClose = () => {
+    setMenuOpen(false);
+  };
+
+  const menuItems = (
+    <List>
+      <ListItem component={Link} to="/" onClick={handleClose}>
+        Home
+      </ListItem>
+      <ListItem component={Link} to="/projects" onClick={handleClose}>
+        Projects
+      </ListItem>
+      <ListItem component={Link} to="/volunteer" onClick={handleClose}>
+        Volunteer
+      </ListItem>
+    </List>
+  );
+
+  return (
+    <AppBar position="static" color={darkMode ? 'dark' : 'primary'}>
+      <Toolbar sx={{ height: '100px', alignItems: 'center' }}>
+        <Box sx={logoBlobStyle}></Box>
+        <Link to="/" style={textStyle} aria-label="Home">
+          <Box
+            component="img"
+            position="relative"
+            alt="CODE PDX logo"
+            src={'/assets/rose_logo.png'}
+            sx={{
+              width: '75px',
+              mr: '30px',
+              ml: { xs: '0px', sm: '30px', lg: '20px' }
+            }}
+          />
+        </Link>
+        <Typography variant="h4" display={{ xs: 'none', sm: 'block' }} pl={'15px'}>
+          <Link to="/" style={textStyle} aria-label="Home">
+            CODE PDX
+          </Link>
+        </Typography>
+
+        {/* Display for larger viewports */}
+        <Typography
+          variant="body1"
+          ml={'auto'}
+          display={{ xs: 'none', sm: 'block' }}
+          marginRight={2}
+        >
+          <Link to="/projects" style={textStyle} aria-label="Projects">
+            Projects
+          </Link>
+        </Typography>
+        <Typography variant="body1" display={{ xs: 'none', sm: 'block' }} marginRight={2}>
+          <Link to="/volunteer" style={textStyle} aria-label="Volunteer">
+            Volunteer
+          </Link>
+        </Typography>
+
+        {/* Hamburger menu for smaller viewports */}
+        <Hidden smUp>
+          <IconButton
+            sx={{ ml: 'auto' }}
+            edge="end"
+            color="inherit"
+            aria-label="menu"
+            onClick={handleToggle}
+            ref={anchorRef}
+          >
+            <MenuIcon />
+          </IconButton>
+        </Hidden>
+        <IconButton edge="end" color="inherit" onClick={handleThemeChange}>
+          {darkMode ? <Brightness7 /> : <Brightness4 />}
```

Comment:
For the dark mode button, maybe add it to the menu for smaller viewports? It might be the least interacted-with button on there. A lot of sites that I've seen place it inside a menu for all viewports.

Reply:
Once I make sure the setup is right It wont exist on mobile.  It will just connect with mobile theme or display brightness. That will come once theme updates are done.